### ISSUE TITLE
Yeelight not found by Homey

### ISSUE THREAD

**Issue by lindblomstefan** [jghaanstra/com.xiaomi-miio#81]
Homey lost my Yeelight gen2 (color). Added it in the official Yeelight app as instructed (as done last time adding it, half a year ago). Enabled Lan Control, went back to Homey (2.0) but the Yeelight is not found. Have restarted Homey, reinstalled your Mi Home App, restarted it, re-added the bulb in official Yeelight app but Homey can’t find it however I do.

**Comment by jghaanstra** [comment]
Do you know how to do a CLI install? I can create a version of the app that will spit out some extra debug logging on the command line.

**Comment by lindblomstefan** [comment]
I’m afraid the answer is no :/

**Comment by jghaanstra** [comment]
Looks like there is a bug in the pairing wizard. I’ll submit a fix this evening.

**Comment by WrDX** [comment]
I'm having an issue with pairing yeelights too, if I can help debugging etc, hit me up on slack (wouterdt). I can CLI install if you need.

**Comment by jghaanstra** [comment]
@WrDX you can try out the current version here on GitHub and see if your issue is resolved.

**Comment by WrDX** [comment]
Sure

**Comment by WrDX** [comment]
I've installed 2.11.4.
It should find 8 new yeelights (version 2, color) but finds none unfortunately.

**Comment by jghaanstra** [comment]
I pushed a commit to Github with some extra debug logging. Could you run it from command line with `athom app run` and post the output here.

**Comment by WrDX** [comment]
```
me@me:~/Projects/com.xiaomi-miio$ athom app run
✓ Pre-processing app...
✓ Validating app...
✓ Homey App validated successfully against level `debug`
✓ Packing Homey App...
 — App size: 4.04 MB
✓ Installing Homey App on `Umbrella Corp.` (http://[IP]:80)...
✓ Homey App `com.xiaomi-miio` successfully installed
✓ Running `com.xiaomi-miio`, press CTRL+C to quit
 — Profile your app's performance at https://go.athom.com/app-profiling?homey=[token]&app=com.xiaomi-miio
─────────────── Logging stdout & stderr ───────────────
2019-03-03 19:57:14 [log] [XiaomiMiioApp] Initializing Xiaomi Mi Home app ...
```

And then nothing, not even when I try to add a device. I restarted the app in the homey app, that gave a log message:

```--- INFO: com.xiaomi-miio has been killed ---
2019-03-03 20:02:09 [log] [XiaomiMiioApp] Initializing Xiaomi Mi Home app ...```

So logging should be working.

ps. can we do this over Slack instead pls?

**Comment by jghaanstra** [comment]
Not right now, but I'll contact you soon.

**Comment by jghaanstra** [comment]
Fixed with release 2.11.4.
